packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -278,7 +278,6 @@ $btn-icon-sizeXSmall-iconSize          : 18px;
     position: relative;
     display: inline-block;
     vertical-align: middle;
-    font-family: $btn-default-font-family;
```

Comment:
Are we sure we don't need this? I mentioned on the related Fozzie PR that, on menu, I had to manually set the font-face value otherwise it rest back to the browser default (body styles dinner seem to apply to buttons).

Reply:
In the latest fozzie update I added font-family to [a button element ](https://github.com/justeat/fozzie/blob/24cc930c798a4e5a97563b6bbe21f52d214ed022/src/scss/base/_typography.scss#L186). This way if consuming app uses fozzie typography buttons will have our font, if not it is up to consuming app what font to specify.

Reply:
Right, gotcha 👍🏻